src/components/InteriorLeftNavList/InteriorLeftNavList.js

```diff
@@ -68,14 +72,15 @@ class InteriorLeftNavList extends Component {
       onListClick, // eslint-disable-line no-unused-vars
       onItemClick, // eslint-disable-line no-unused-vars
       activeHref, // eslint-disable-line no-unused-vars
+      lockOpen,
       ...other
     } = this.props;
 
     const classNames = classnames(
       'left-nav-list__item',
       'left-nav-list__item--has-children',
       {
-        'left-nav-list__item--expanded': this.state.open,
+        'left-nav-list__item--expanded': lockOpen || this.state.open,
```

Comment:
Hi @ehdsouza nice addition! - Given this PR adds logic to prevent `state` from being changed, ignoring the state here looks redundant?

Reply:
👍 Good catch, Ill make the change

Reply:
Great thanks @ehdsouza!